server/src/service-layer/controllers/AdminController.ts

```diff
@@ -249,6 +253,42 @@ export class AdminController extends Controller {
     }
   }
 
+  @SuccessResponse("200", "User found")
+  @Get("/users/{uid}")
+  public async getUser(@Path() uid: string): Promise<GetUserResponse> {
+    try {
+      const userService = new UserDataService()
+      const user = await userService.getUserData(uid)
+
+      if (!user) {
+        this.setStatus(404)
+        return { error: "User not found" }
+      }
+
+      const authService = new AuthService()
+      const userAuthData = await authService.retrieveUserByUid(uid)
+      const { customClaims, email, metadata } = { ...userAuthData }
+
+      this.setStatus(200)
+      return {
+        data: {
+          email,
+          membership: customClaims?.[AuthServiceClaims.ADMIN]
+            ? UserAccountTypes.ADMIN
+            : customClaims?.[AuthServiceClaims.MEMBER]
+              ? UserAccountTypes.MEMBER
+              : UserAccountTypes.GUEST,
```

Comment:
Nit: We've done this so many times already - might be worth creating a tested method in `AuthService` for it (can possible be another ticket)